### PR TITLE
fix(ui): fix search not working in roles input  in user profile page

### LINKED ISSUE #13895: Search in Roles input is not working in User Profile Page
**Affected module**
Does it impact the UI, backend or Ingestion Framework?
UI

**Describe the bug**

- Search in Roles input is not working in User Profile Page

**To Reproduce**

Screenshots or steps to reproduce

https://github.com/open-metadata/OpenMetadata/assets/66266464/c8e525a4-cf12-411f-b359-153c044e9d6f

**Expected behavior**

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

Fixes #13895

- I worked on fixing the search not working in roles input  in the user profile page

https://github.com/open-metadata/OpenMetadata/assets/66266464/91bc371f-c6ea-4849-85a5-62b8223141ef

#
### Type of change:

- [x] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx [hunk 1]
```diff
@@ -868,3 +869,17 @@ export const getUniqueArray = (count: number) =>
...
+ * @param option select options list
+ * @returns boolean
+ */
+export const handleSearchFilterOption = (
+  searchValue: string,
+  option?: {
+    label: string;
+    value: string;
+  }
+) => lowerCase(option?.label).includes(lowerCase(searchValue));
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx:884]: The `lowerCase` will convert the string into space separated lowercase string. so instead use `toLowerCase()` native sting method.

example:

```js
lowerCase('fooBar');
// => 'foo bar'
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
Left a few comments, thanks

**Comment by Ashish8689** [comment]
@Sachin-chaurasiya  changes made as per comments!

File: openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx (hunk 1 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx]: [toLower](https://lodash.com/docs/4.17.15#toLower) is also an option here

**Review by chirag-madlani — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**